docs/sample_config.yaml

```diff
@@ -1657,7 +1657,19 @@ email:
   #template_dir: "res/templates"
 
 
-#password_providers:
+# Password providers allow homeserver administrators to integrate
+# their Synapse installation with existing authentication methods
+# ex. LDAP, external tokens, etc.
+#
+# For more information and known implementations, please see
+# https://github.com/matrix-org/synapse/blob/master/docs/password_auth_providers.md
+#
+# Note: instances wishing to use SAML or CAS authentication should
+# instead use the `saml2_config` or `cas_config` options,
+# respectively.
+#
+password_providers:
+#    # Example config for an LDAP auth provider
```

Comment:
I think this breaks the rule in the docs about a whole dict being commented out:

> Where several settings are grouped into a single dict, avoid the convention where the whole block is commented out, resulting in comment lines starting `# #` ...

Reply:
(Granted, this PR doesn't make the situation worse...)

Reply:
I think by whole block it means the `password_providers` bit.

An example of a good block: https://github.com/matrix-org/synapse/blob/ec5ac8e2b129f645a06f441143c2dcd2fb1c7037/docs/sample_config.yaml#L1892-L1929